contributors/devel/sig-testing/bazel.md

```diff
@@ -2,22 +2,22 @@
 
 Building and testing Kubernetes with  Bazel is supported but not yet default.
 
-Bazel is used to run all Kubernetes PRs on [Prow](https://prow.k8s.io),
+Bazel is used to run all Kubernetes PRs on [Prow],
 as remote caching enables significantly reduced build and test times.
 
 Some repositories (such as kubernetes/test-infra) have switched to using Bazel
 exclusively for all build, test, and release workflows.
 
-Go rules are managed by the [`gazelle`](https://github.com/bazelbuild/rules_go/tree/master/go/tools/gazelle)
-tool, with some additional rules managed by the [`kazel`](https://git.k8s.io/repo-infra/kazel) tool.
+Go rules are managed by the [`gazelle`][gazelle]
+tool, with some additional rules managed by the [`kazel`][kazel] tool.
 These tools are called via the `hack/update-bazel.sh` script.
 
-Instructions for installing Bazel
-can be found [here](https://www.bazel.io/versions/master/docs/install.html).
-Please note that until [this Bazel
-issue](https://github.com/bazelbuild/rules_docker/issues/454) is fixed,
-`/usr/bin/env python` must be python2 in order for all the Bazel commands listed
-below to succeed.
+Instructions for installing Bazel can be found [here][bazel-install].
+Note that Bazel currently does not work with Python 3, which means that
```

Comment:
I do not believe in the accuracy of this statement. Why do you believe it?

If it is true, what issues should we track (please link to them) that will help us understand that this documentation is stale.

Reply:
https://github.com/bazelbuild/bazel/issues/8443 (referenced in the PR description too), which has since been fixed. I’m not sure if something needs to be done to get that fix into Kubernetes too (how does the release tooling determine which Bazel version to use?).

Reply:
Bazel supports python**3** via toolchain since bazel 0.27.0. Here is a good FAQ (bazelbuild/bazel#7899)

Moreover, the default `python_version` for bazel was changed to python**3** in [0.25.0](https://github.com/bazelbuild/bazel/blob/master/CHANGELOG.md#release-0250-2019-05-01) (bazelbuild/bazel#7359) and `--python_path` is deprecated as of [0.27.0](https://github.com/bazelbuild/bazel/blob/master/CHANGELOG.md#release-0270-2019-06-17)

> Python rules now determine the Python runtime using toolchains rather than `--python_top` and `--python_path`, which are deprecated. See [#7899](https://github.com/bazelbuild/bazel/issues/7899) for information on declaring Python toolchains and migrating your code. As a side-benefit, this addresses [#4815](https://github.com/bazelbuild/bazel/issues/4815) (incorrect interpreter version used) on non-Windows platforms. Note however that some builds break due to getting the version they asked for -- consider setting `python_version = "PY2"` on Python 2 targets and `--host_force_python=PY2` if any Python 2 targets are used in the host configuration. You can temporarily opt out of this change with `--incompatible_use_python_toolchains=false`.

Reply:
Thanks! I've rewritten the paragraph and included a link to the FAQ as well.